shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -792,22 +810,20 @@ private void finishFrame(boolean isFrameRenderedUsingImageReaders) {
       }
     }
 
-    for (int i = 0; i < platformViews.size(); i++) {
-      int viewId = platformViews.keyAt(i);
-      View platformView = platformViews.get(viewId);
-      View mutatorView = mutatorViews.get(viewId);
+    for (int i = 0; i < platformViewParent.size(); i++) {
+      final int viewId = platformViewParent.keyAt(i);
+      final View parentView = platformViewParent.get(viewId);
```

Comment:
Is it possible parentView will be null?

Reply:
It's not possible. This is iterating over the existing entries, and the code never puts `null` in the dictionary.